wlroots/ffi_build.py

```diff
@@ -259,6 +259,11 @@
     ...;
 };
 
+enum wl_keyboard_key_state {
+    WL_KEYBOARD_KEY_STATE_RELEASED = 0,
+    WL_KEYBOARD_KEY_STATE_PRESSED = 1,
+};
```

Comment:
Hmmm, this is a part of Wayland, but I don't think we bind the C enum directly, rather we define the `key_state` `enum.Enum` and then do the marshaling / unmarshaling of the enum value. Is this something that we could move to Wayland or make use of the enum that is currently defined there?

Reply:
Yeah think you're right, using the enum already generated by the scanner makes the most sense, and is probably a bit more future proof in case it is changed in the future by upstream